src/index.html

```diff
@@ -1,13 +1,54 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
-</html>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+  </head>
+  <body class="body">
+    <div class="section" data-qa="card">
+      <ul class="product-cards section__list">
+        <li class="product-cards__items">
+
+          <div class="image product-cards__image">
+            <img
+          src="images/imac.jpeg"
+          alt="Apple A1419"
+          class="image__item"
+          >
+          </div>
+          <h1 class="product-cards__text">
+            Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)
+          </h1>
+          <p class="product-cards__article">Код товара: 195434</p>
```

Comment:
It's not an article, so element name should be different.
```suggestion
          <p class="product-cards__code">Код товара: 195434</p>
```